packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js

```diff
@@ -1,13 +1,20 @@
-'use strict'
+`use strict`
```

Comment:
?  You aren't defaulting to all template literals, so why change this one?

Reply:
Changed